literature.bib

```diff
@@ -371,6 +396,21 @@ @inproceedings{DHK:SPLC24
 	doi = {10.1145/3646548.3676538}
 }
 
+@inproceedings{PKT+:SPLC24,
+	author = {Pett, Tobias and Krieter, Sebastian and Th\"{u}m, Thomas and Schaefer, Ina},
+	title = {{MulTi-Wise Sampling: Trading Uniform T-Wise Feature Interaction Coverage for Smaller Samples}},
+	year = 2024,
+	isbn = {9798400705939},
+	publisher = ACM,
+	address = NY,
+	doi = {10.1145/3646548.3672589},
+	booktitle = SPLC,
+	pages = {47–53},
```

Comment:
Wrong - symbol I think.

Reply:
Yes, use `--`. ACM wrongly uses the unicode dash in the BibTeX entries from their website, which is annoying.